aws/plugin.go

```diff
@@ -334,7 +334,6 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"aws_oam_sink":                                                 tableAwsOAMSink(ctx),
 			"aws_opensearch_domain":                                        tableAwsOpenSearchDomain(ctx),
 			"aws_organizations_account":                                    tableAwsOrganizationsAccount(ctx),
-			"aws_organizations_organizational_unit":                        tableAwsOrganizationsOrganizationalUnit(ctx),
```

Comment:
Can we comment this out instead?

Reply:
I added back the removed code and commented it.